### ISSUE TITLE
add auto publish action

### ISSUE THREAD

**Issue by cedvdb** [cedvdb/phone_numbers_parser#73]
There is a github action that updates the metadata in this github repository. 

It would be nice to have the package be submitted to pub.dev without my manual intervention once this is done.

**Comment by ManindraDeMel** [comment]
Hi please assign me the ticket

**Comment by ManindraDeMel** [comment]
Hi sorry, im available to look at the previous issue from PR #78
